base/sysinfo.jl

```diff
@@ -141,12 +141,22 @@ function cpu_info()
     return cpus
 end
 
+"""
+    uptime()
+
+Gets the current system uptime.
```

Comment:
Ok so to clarify the only reason that `MACHINE` `KERNEL` and other exported API in the `Sys` module  *include* the `Sys` prefix, e.g. `Sys.KERNEL`,  is because this is the expected usage?
How do I then know if `uptime` or `loadavg` will need the `Sys` prefix? They too are exported API.

Might not a simple rule: if exported don't include the module prefix otherwise if unexported include it, be simplest and clearest?